website/pages/home/index.jsx

```diff
@@ -112,78 +125,102 @@ export default function HomePage() {
-            ]}
-          />
-        </div>
-      </div>
+              {
+                alt: 'AWS',
+                url: require('./img/logos/aws-color.svg?url'),
+              },
+              {
+                alt: 'Unknown',
+                url: require('./img/logos/unknown.svg?url'),
+              },
+              {
+                alt: 'MySQL',
+                url: require('./img/logos/my-sql-color.svg?url'),
+              },
+              {
+                alt: 'More integrations',
+                url: require('./img/logos/custom.svg?url'),
+              },
+            ],
+          },
+        ]}
+      />
+
+      <section className="why-boundary">
+        <WhyBoundary
+          heading="Why Boundary"
+          items={[
+            {
+              icon: require('./img/icons/code-white.svg?url'),
+              heading: 'On-demand identity-based access to infrastructure',
+              description:
+                'Securely connect trusted identities to logical services without having to create or store credentials or access.',
+            },
+            {
+              icon: require('./img/icons/layers-white.svg?url'),
+              heading: 'Scale access management in dynamic environments',
+              description:
+                'Scale access management by defining access controls around logical services instead of IP-based access policies.',
+            },
+            {
+              icon: require('./img/icons/refresh-white.svg?url'),
+              heading:
+                'Streamline remote access with easy-to-use CLI, API or UI',
+              description:
+                'Make it easy to access all applications and systems through a single workflow that works with existing tooling.',
+            },
+          ]}
+        />
+      </section>
 
-      <div className="break-section">
-        <SectionBreakCta
-          heading="Have you tried Boundary?"
-          content="Share your feedback for a chance to receive special swag."
-          link={{
-            text: 'Share your Boundary story',
-            url: 'http://hashi.co/boundary-survey',
+      <section className="how-boundary-works">
+        <HowBoundaryWorks
+          heading="How Boundary Works"
+          description="HashiCorp Boundary is a secure remote access solution that provides an easy way to allow access to applications and critical systems with fine-grained authorizations based on trusted identities. Across clouds, local data centers, low-trust networks, Boundary provides an easier way to protect and safeguard access to application and critical systems by trusted identities without exposing the underlying network"
+          items={[
+            'Platform-agnostic proxy for dynamic targets',
+            'No SSH keys or VPN credentials to manage',
+            'Just-in-time credentials via HashiCorp Vault',
+          ]}
+          img={{
+            src: require('./img/how-boundary-works.png'),
```

Comment:
This one too is a little blurry; curious if we can export as an SVG here?

Reply:
This is actually a screenshot in the design file, so I wasn't able to convert to SVG.

Reply:
Got it https://github.com/hashicorp/boundary/pull/1558/commits/f7dbdb60a18a34b92761562eec33b37729644da2